terraform/deployments/govuk-publishing-platform/app_router.tf

```diff
@@ -78,7 +78,7 @@ module "router" {
       # BACKEND_URL_service-manual-frontend = module.service-manual-frontend.virtual_service_names[0]
       # BACKEND_URL_smartanswers            = module.smartanswers.virtual_service_names[0]
       # BACKEND_URL_spotlight               = module.spotlight.virtual_service_names[0]
-      BACKEND_URL_static = module.static.virtual_service_names[0]
+      BACKEND_URL_static = "http://${module.static.virtual_service_names[0]}"
```

Comment:
I guess with https://github.com/alphagov/govuk-infrastructure/pull/226, the array needs to be removed.